sdk/state/integration/helpers_test.go

```diff
@@ -93,63 +127,127 @@ func initAccounts(t *testing.T) (initiator Participant, responder Participant) {
 		txResp, err := client.SubmitFeeBumpTransaction(fbtx)
 		require.NoError(t, err)
 		responder.EscrowSequenceNumber = int64(txResp.Ledger) << 32
+
+		// add initial contribution
+		_, err = account.IncrementSequenceNumber()
+		require.NoError(t, err)
+
+		tx, err = txnbuild.NewTransaction(txnbuild.TransactionParams{
```

Comment:
seems very similar to the section above around line 67. If there's enough similarity and they are serving a similar purpose, would it make sense to extract into a function?

Reply:
yup you're right, made another helper method for creating the escrow account that removed the dup code
https://github.com/stellar/experimental-payment-channels/pull/86/commits/d2c2c737ad46924bc6f929024fb143fa441ddf5a